release-notes/DIS/DIS_3.1.md

```diff
@@ -4,6 +4,102 @@ uid: DIS_3.1
+- [Skyline.DataMiner.CICD.Validators.Protocol](https://www.nuget.org/packages/Skyline.DataMiner.CICD.Validators.Protocol) version 2.0.0
+- [Skyline.DataMiner.CICD.Validators.Common](https://www.nuget.org/packages/Skyline.DataMiner.CICD.Validators.Common) version 2.0.0
+- [Skyline.DataMiner.CICD.Models.Protocol](https://www.nuget.org/packages/Skyline.DataMiner.CICD.Models.Protocol) version 1.0.16
+- [Skyline.DataMiner.Core.ArtifactDownloader](https://www.nuget.org/packages/Skyline.DataMiner.Core.ArtifactDownloader) version 3.1.1
+- [Skyline.DataMiner.Dev.Common](https://www.nuget.org/packages/Skyline.DataMiner.Dev.Common) version 10.5.10
+- [Skyline.DataMiner.Dev.Automation](https://www.nuget.org/packages/Skyline.DataMiner.Dev.Automation) version 10.5.10
+- [Skyline.DataMiner.Dev.Protocol](https://www.nuget.org/packages/Skyline.DataMiner.Dev.Protocol) version 10.5.10
+- [Skyline.DataMiner.XmlSchemas](https://www.nuget.org/packages/Skyline.DataMiner.XmlSchemas) version 1.1.4
+
+### Fixes
+
+#### IDE
+
+##### Problem when an operation failed because of insufficient permissions [ID 43538]
+
+Up to now, a fatal error would occur when an operation was performed for which insufficient permissions were granted on the destination DMA. From now on, a message will appear, indicating that the operation failed because of insufficient permissions.
+
+Also, error popups will now reliably appear in front of Visual Studio. They will no longer appear behind the main window. In addition, the appearance and consistency of error popups have been aligned.
+
+##### Problem when selecting NuGet references in Solution Explorer [ID 43544]
+
+Up to now, selecting a NuGet reference in Visual Studio's Solution Explorer (within a package solution) could cause an infinite loop of error popups, eventually leading to a fatal error in Visual Studio.
+
+##### Skyline.DataMiner.Sdk projects would incorrectly not allow multiple Exe blocks regardless of their type [ID 43784]
+
+With the introduction of the Skyline.DataMiner.Sdk projects, Script style projects would incorrectly not allow multiple [Exe](xref:DMSScript.Script.Exe) blocks, regardless of their [type](xref:DMSScript.Script.Exe-type).
+
+From now on, Script style projects will not allow multiple Exe blocks of type 'csharp'. This will allow users to have e.g. an Exe block of type 'csharp' and an Exe block of type 'report'.
+
+##### ProcessAutomation.dll would be referenced incorrectly after building a dmapp [ID 43899]
+
+Up to now, when you had created or deployed an Automation script that referenced the *ProcessAutomation.dll* file, that assembly would be resolved incorrectly. The system would attempt to locate it in the `ProtocolScripts/DllImport` folder, whereas the correct location was `ProtocolScripts`.
+
+From now on, when the *ProcessAutomation.dll* file is referenced, the resulting path in the XML file will point to the `ProtocolScripts` folder.
+
+##### Removed Automation script interactivity check
```

Comment:
@PedroDebevere , no RN ID was specified here. Can you check which RN should be indicated here?

Reply:
> @PedroDebevere , no RN ID was specified here. Can you check which RN should be indicated here?

RN43899

Reply:
I only have a single suggestion, other than that, I have no remarks

Reply:
As to that missing RN ID, I meant **Removed Automation script interactivity check** :-)